clients/ui/docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+services:
+  frontend:
+    build: ./frontend
+    container_name: model-registry-ui
+    ports:
+      - 8080:8080
+    environment:
+      API_URL: http://model-registry-bff:4001
+    networks:
+      - model_registry
+  bff:
```

Comment:
```
    depends_on:
      - bff
```

nit: it can be a good practice to start up the frontend after the backend to avoid reloads.